frontend/src/metabase/visualizations/visualizations/CartesianChart/use-models-and-option.ts

```diff
@@ -49,12 +52,12 @@ export function useModelsAndOption({
 
   const renderingContext: RenderingContext = useMemo(
     () => ({
-      getColor: color,
+      getColor: name => color(name, palette),
```

Comment:
passing colors we want to override here looks good. I kind of remember we did this when we worked with static viz to make user appearance colors works.